test/lib/analyzer/os-release-detector.test.ts

```diff
@@ -218,6 +221,37 @@ test("os release detection", async (t) => {
   }
 });
 
+test("os release detection statically", async (t) => {
+  const examples = {
+    "centos:7": {
+      dir: "centos",
+      imageType: ImageType.DockerArchive,
+      imagePath: path.join(
+        __dirname,
+        "../../fixtures/docker-archives/skopeo-copy/centos-6.tar",
+      ),
+      expected: { name: "centos", version: "6", prettyName: "" },
```

Comment:
The key is named centos:7 but the expected is centos version 6? 😕

Reply:
My bad! It was copy and paste! It didn't interfere in the test itself. It just a key. I'm changing it!